Build.PL

```diff
@@ -39,15 +39,15 @@ my $build = WTSI::DNAP::Utilities::Build->new
                           'IPC::Run'                  => '>= 0.92',
                           'JSON'                      => '>= 2.90',
                           'Log::Log4perl'             => '>= 1.46',
-                          'List::AllUtils'            => '>= 0.09',
+                          'List::MoreUtils'           => '0',
                           'Moose'                     => '>= 2.1',
                           'MooseX::StrictConstructor' => '>= 0.19',
                           'MooseX::Types'             => '>= 0.45',
                           'Readonly'                  => '>= 1.04',
                           'Set::Scalar'               => '>= 1.29',
                           'Try::Tiny'                 => '>= 0.22',
                           'URI'                       => '>= 1.67',
-			  'UUID'                      => '>= 0.27',
+			                    'UUID'                      => '>= 0.27',
```

Comment:
Convert tab to space to fix indentation?

Reply:
Fixed - my editor was hiding that.